roles/openshift_health_checker/library/docker_info.py

```diff
@@ -0,0 +1,23 @@
+# pylint: disable=missing-docstring
+"""
+Ansible module for determining information about the docker host.
+"""
+
+from ansible.module_utils.docker_common import AnsibleDockerClient
+
+
+def main():
+    client = AnsibleDockerClient(
+        argument_spec=dict(
+            name=dict(type='list'),
+        ),
+    )
+
+    client.module.exit_json(
+        changed=False,
+        info=client.info(),
+    )
+
+
+if __name__ == '__main__':
+    main()
```

Comment:
Would it make sense to make this available to other roles? I could see this being of value in the docker-related roles as well. It might also be nice to return back at least some of the values as 'facts' instead of just a single `info` dict.